src/components/Button/Button.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../styles/variables.scss';
+
+.Palmetto-Button {
+  position: relative;
+  padding: $size-spacing-sm $size-spacing-md;
+  border: none;
+  border-radius: 8px;
+  color: $color-base-white;
+  background-color: #2F8459; // $color-brand-primary-base;
+  font-weight: 700;
+  letter-spacing: .2px;
```

Comment:
Should we make letter spacing global? Or does the button require something unique?

Reply:
@nathanyoung added this and it was to deal with letters getting too crowded when they have a bold font weight, I believe.

Reply:
don't want to make it global, what @JCarmine said is correct. i added the spacing because the 700 weight font needs some spacing between the letters to help with legibility.

Reply:
If we haven't implemented our actual base font yet, should we be getting this granular with styling?

Reply:
it is here: https://github.com/palmetto/palmetto-components/blob/develop/src/styles/reset.scss#L2